### DISCUSSION TITLE
WW3 error in WMGLOW

### DISCUSSION THREAD

**Discussion by famarti** [NOAA-EMC/WW3#642] [Q&A] [answered]
Hi everyone! I'm trying to run the model (latest version) with 5 different grids, and when it reaches the ww3_multi program, I got the following errors:

 WAVEWATCH III ERROR IN WMGLOW : 
     CANNOT FIND SOURCE FOR BOUNDARY DATA
     GRID, IX, IY, X, Y:     2   588   109  0.1135E+03 -0.2500E+02
 WAVEWATCH III ERROR IN WMGLOW : 
     CANNOT FIND SOURCE FOR BOUNDARY DATA
     GRID, IX, IY, X, Y:     2   210   129 -0.7550E+02 -0.1500E+02
  WAVEWATCH III ERROR IN WMGLOW : 
     CANNOT FIND SOURCE FOR BOUNDARY DATA
     GRID, IX, IY, X, Y:     2   283   129 -0.3900E+02 -0.1500E+02
...
(and many more like those).

I tried to read the source code but it's quite massive, and hard to follow. Maybe its something like an "out of bounds" error? 
Any suggestions on this?

**Comment by aliabdolali** [comment, marked as answer]
Hi @famarti it is related to your setup. You need to take a look at your mask file in GRID no 2. It seems those indexes are defined as open boundary conditions in the mask (with 2), but it cannot find boundary data in the parent grid (no 1).

    0 : Land point
    1 : Sea point
    2 : Active boundary point
    3 : Excluded point

**Reply by famarti** [reply]
Thank you so much for your answer, Ali! You were right about the mask file.
